cmd/main.go

```diff
@@ -84,6 +90,65 @@ func main() {
 		l.Fatalf("Enclave terminated: %v", err)
 	}
 
-	// Block on this read forever.
-	<-make(chan struct{})
+	// Nitriding supports two ways of starting the enclave application:
+	//
+	// 1) Nitriding spawns the enclave application itself, and waits for it
+	//    to terminate.
+	//
+	// 2) The enclave application is started by a shell script (which also
+	//    starts nitriding).  In this case, we simply block forever.
+	if appCmd != "" {
+		f := func(s string) {
+			l.Printf("Application says: %s", s)
```

Comment:
Oh, does this format string need to have a terminating `\n` since `Scanner` absorbs them?

Reply:
Good catch but no: Go's [Logger package](https://pkg.go.dev/log#Logger.Output) (which we are using here) is automatically appending a newline to log messages:
> A newline is appended if the last character of s is not already a newline.